### DISCUSSION TITLE
useChain to trigger some animations sequentially and others simultaneously?

### DISCUSSION THREAD

**Discussion by benwerner01** [pmndrs/react-spring#1098] [Support]
I'm using `useChain` to sequentially trigger a couple animations (let's say they're `A1`, `A2`, `A3`, and `A4`) and it's working fine. However what I really want is to trigger `A1`, then trigger `A2` **and** `A3` simultaneously, and then trigger `A4`. Is this not currently possible using react-spring, or am I missing something? Thanks!

**Comment by jvfresco** [comment]
I'm going to ask this question in case someone uses the search like I did.

With useChain you have the option to define timeSteps (check the docs)
In your example, you could do:
`useChain([A1,A2,A3,A4], [0,1,1,2])`
A1 would be animated immediately (it has a timeStep 0), A2 and A3 would be animated after 1 second simultaneously, and A4 would be animated after 2 seconds

**Reply by benwerner01** [reply]
That's what I was looking for, thanks!
